### ISSUE TITLE
automatic1111 won't start up

### ISSUE THREAD

**Issue by mdlieber99** [TheLastBen/fast-stable-diffusion#2818]
[UPDATE: clicking this option: "Use_Cloudflare_Tunnel:" got it working again, but the previews take forever to show up on the gui screen. does this help narrow down the problem?]

I was using automatic1111 on google colab just fine yesterday morning, but now it hangs during the final stage of the start-up process. I was using an A100 processor.  Here is the output I get:

ControlNet v1.1.183
ControlNet v1.1.183
Loading weights [3e9ec5fd17] from /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/Nicola_Anna.safetensors
Creating model from config: /content/gdrive/MyDrive/sd/stable-diffusion-webui/configs/v1-inference.yaml
Applying attention optimization: xformers... done.
Model loaded in 4.6s (load weights from disk: 0.5s, create model: 0.8s, apply weights to model: 2.4s, load textual inversion embeddings: 0.5s, calculate empty prompt: 0.2s).

but then it never proceeds past this point.  in one of the many attempts to get this to work (including after restarting my Mac), it told me I needed to manually add "frpc_linux_amd64_v0.2" to the radio folder, which I did.  Now it is back to just hanging indefinitely with just the output above

**Comment by AYKIQ** [comment]
yes, Ihave it too

**Comment by mdlieber99** [comment]
clicking this option: "Use_Cloudflare_Tunnel:" got it working again, but the previews take forever to show up on the gui screen.  does this help narrow down the problem?

**Comment by yasu5555** [comment]
I have the same problem

**Comment by global-offer** [comment]
I have the same problem. Please help me.

**Comment by mr-mitt** [comment]
i had this problem earlier today and it was because Gradio server was down. Maybe again. Just tried starting SD and is also hanging at the end. check here... Im going to bed!!
https://status.gradio.app/793595965#logs

**Comment by Azura-13** [comment]
Same here but have had this problem for days
(Google Colab Pro)

**Comment by Giusemoli312** [comment]
Same problem here, for days now..

**Comment by KyotoLyon** [comment]
Same issue here, 2 days and still no solution?

**Comment by mdlieber99** [comment]
For everyone having problems, this worked for me:

click this option: "Use_Cloudflare_Tunnel:" when opening automatic1111

does it work for you too?

**Comment by alexjl90210** [comment]
yes same issue but "Use_Cloudflare_Tunnel:" does work now too      It was working without the box checked earlier today.

**Comment by Maximb1985** [comment]
> For everyone having problems, this worked for me:
> 
> click this option: "Use_Cloudflare_Tunnel:" when opening automatic1111
> 
> does it work for you too?

How to select this function(Use_Cloudflare_Tunnel:) please tell me?
